### PR TITLE
Use Argparse and add options to not include hlines

### PR DESCRIPTION
**Pull request by BenBals**

### CODE REVIEW

File: truthtable.py [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+import argparse
 
-if len(sys.argv) > 1 and sys.argv[1] != "-":
+parser = argparse.ArgumentParser(description='A LaTeX truthtable generator')
+parser.add_argument("inputFile", help="File from which the logical expression is read.")
+parser.add_argument("--nohlines", dest="nohlines", default=False, const=True, nargs="?", help="Don't put seperatiing horizontal lines in the output table.")
+
+args = parser.parse_args()
+
+if args.inputFile:
```
Comment (felixrindt) [inline truthtable.py]: could you keep the condition intact and also check for file name not being `-`?

**Review by felixrindt — CHANGES_REQUESTED** [review]

File: truthtable.py (hunk 1 above)
Comment (BenBals) [reply truthtable.py]: I have changed it to include the check for "-" but I am not sure what you mean by keeping the condition intact. (I have never done python before.)

Comment (felixrindt) [reply truthtable.py]: I just meant that we shouldn't change the condition. LGTM now

**Comment by felixrindt** [comment]
Thanks :)

**Pull request merged**